src/ContentScript/injectors/youtube.plugin.ts

```diff
@@ -1,10 +1,67 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-unused-vars */
+// import { debounce } from 'lodash';
+import LikeButton from '../../sdk/button.iframe';
+
 class YoutubePlugin {
+  youtubeStyle!: HTMLElement;
+
   constructor() {
     console.log('workd');
   }
 
+  insertStyle() {
+    this.youtubeStyle = document.createElement('style');
+    this.youtubeStyle.innerHTML = `
+              #meta-contents {
+                display: flex;
+                flex-direction: row;
+                justify-content: space-between;
+              }
+          `;
+    document.body.appendChild(this.youtubeStyle);
+  }
+
   inject() {
     console.log(this);
+    this.onPageLoaded();
+  }
+
+  private onPageLoaded() {
+    if (!document.querySelector('ytd-app')) {
+      setTimeout(() => {
+        this.onPageLoaded();
+      }, 500);
+    } else {
+      const id = this.getLikeId() || 'foundation';
+      this.insertLikeCoinButton(id);
+    }
+  }
+
+  private getLikeId() {
+    const desc = document.querySelector('#description');
+    console.log(desc, this);
+    const nodes = desc?.querySelectorAll('a') as NodeListOf<HTMLAnchorElement>;
+    console.log(nodes);
+    if (nodes.length === 0) return;
+    const node = nodes[nodes.length - 1];
+    const url = node.innerText;
+    console.log(url);
+    const id = url.split('/')[url.split('/').length - 1] || 'likecoin';
```

Comment:
Here `likecoin` was used as a default liker ID, however in line 36 above `foundation` is used.
Since we will always return `likecoin` from this function, this make the one in line 36 useless, should we just remove this instead?